pycharm.as

```diff
@@ -0,0 +1,9 @@
+def des(n):
+    if n==0 or n==1:
+        print(n, end=" ")
+        return
+    dews(int(n/2))
+    des(n%2)
+    num = int(input("enter any decimal number"))

```

Comment:
Remember to indent the code properly... Take this line outside of the function `des`.